src/display.c

```diff
@@ -602,6 +602,41 @@ void al_acknowledge_drawing_resume(ALLEGRO_DISPLAY *display)
    }
 }
 
+/* Function: al_get_render_state
+ */
+int al_get_render_state(ALLEGRO_RENDER_STATE state)
+{
+   ALLEGRO_DISPLAY *display = al_get_current_display();
+
+   if (!display)
+      return 0;
+
+   switch (state) {
+      case ALLEGRO_ALPHA_TEST:
+         return display->render_state.alpha_test;
+         break;
+      case ALLEGRO_WRITE_MASK:
+         return display->render_state.write_mask;
+         break;
+      case ALLEGRO_DEPTH_TEST:
+         return display->render_state.depth_test;
+         break;
+      case ALLEGRO_DEPTH_FUNCTION:
+         return display->render_state.depth_function;
+         break;
+      case ALLEGRO_ALPHA_FUNCTION:
+         return display->render_state.alpha_function;
+         break;
+      case ALLEGRO_ALPHA_TEST_VALUE:
+         return display->render_state.alpha_test_value;
+         break;
+      default:
+         ALLEGRO_WARN("unknown state to retrieve: %d\n", state);
```

Comment:
We usually capitalize these.

Also, this should be ALLEGRO_ERROR.

Reply:
✅ done

There was another uncapitalized message in `al_set_render_state` I fixed as well in b1a67f8.